generated/nidmm/nidmm/session.py

```diff
@@ -530,6 +530,8 @@ def __init__(self, repeated_capability_list, vi, library, encoding, freeze_it=Fa
         param_list.append("encoding=" + pp.pformat(encoding))
         self._param_list = ', '.join(param_list)
 
+        # Instantiate any repeated capability objects
```

Comment:
This comment makes no sense when no repeated capabilities are found. This is pre-existing and not terribly important. But maybe an easy fix in own PR.